monasca/requirements.lock

```diff
@@ -13,6 +13,6 @@ dependencies:
   version: 0.5.3
 - name: zookeeper
   repository: http://monasca.io/monasca-helm/
-  version: 0.3.3
```

Comment:
dont ever have to manually update this file

Reply:
Done